scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -45,13 +46,34 @@ export function AuthoringTopbarDirective(
             };
 
             scope.openPublishOrSendToPane = () => {
+                const availableTabs = getAvailableTabs();
+                const activeTab = getActiveTab(availableTabs);
+
                 dispatchInternalEvent('interactiveArticleActionStart', {
                     items: [scope.item],
-                    tabs: ['send_to', 'publish'],
-                    activeTab: scope.item.flags?.marked_for_not_publication ? 'send_to' : 'publish',
+                    tabs: availableTabs,
+                    activeTab: activeTab,
                 });
             };
 
+            function getAvailableTabs(): Array<IArticleActionInteractive> {
+                if (scope.isCorrection(scope.item)) {
+                    return ['send_to', 'correct'];
+                } else {
+                    return ['send_to', 'publish'];
+                }
+            }
+
+            function getActiveTab(availableTabs: Array<IArticleActionInteractive>): IArticleActionInteractive {
+                if (availableTabs.includes('correct') && scope.isCorrection(scope.item)) {
```

Comment:
If `scope.isCorrection(scope.item)` is `false`, tab should not be **available**. And if tab is not available it can't be active. You should move that check to `getAvailableTabs` function.